lib/TraackrApi.php

```diff
@@ -92,6 +94,20 @@ public static function setCustomerKey($key) {
    } // End function setCustomerKey()
 
 
+   public static function setExtraHeaders($headers) {
```

Comment:
Good idea. I will probably even go 1 step further, if it's a string we can use it as a single header and an array as multiple headers. Everything else we can return `false`